### PR TITLE
4.0 [BUG] Fix for orderByRaw and inRandomOrder in orderLogic.

### PR DESCRIPTION
**Pull request by kritsot**

### CODE REVIEW

**Comment by kritsot** [comment]
When using raw orderby e.g $this->crud->query->orderByRaw the list is not loaded and an error message is shown.
The error shown in the log is

Undefined index: column at vendor/backpack/crud/src/app/Http/Controllers/Operations/ListOperation.php:116

File: src/app/Http/Controllers/Operations/ListOperation.php [hunk 1]
```diff
@@ -113,6 +113,10 @@ public function search()
         $orderBy = $this->crud->query->getQuery()->orders;
         $hasOrderByPrimaryKey = false;
         collect($orderBy)->each(function ($item, $key) use ($hasOrderByPrimaryKey) {
+            if (isset($item['type']) && $item['type']=='Raw'){
```
Comment (pxpm) [inline src/app/Http/Controllers/Operations/ListOperation.php]: if(!isset($item['column')) {

Comment (kritsot) [reply src/app/Http/Controllers/Operations/ListOperation.php]: sure @pxpm, just pushed the change

**Comment by tabacitu** [comment]
Thank you so much @kritsot & @pxpm . Great collaboration here. I've now merged this into ```master```, it'll be available as 4.0.15 later today.

Cheers!

**Pull request merged**